### PR TITLE
[WIP] New AddBlock component

### PR DESCRIPTION
**Pull request by broskoski**
Getting some work in progress up here. Getting this ready for new onboarding flow.

### CODE REVIEW

File: react/components/AddBlock/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import { isURL } from 'validator';
+
+const Outline = styled.div`
+  width: ${Styles.Constants.blockWidth};
+  height: ${Styles.Constants.blockWidth};
+  background-color: ${Styles.Colors.gray.semiLight};
+  border-radius: .25em;
+  padding: 5px;
+  box-sizing: border-box;
+`;
```
Comment (dzucconi) [inline react/components/AddBlock/index.js]: We got a mixin for this. The trick is that done this way it'll mess up alignment. The mixin just avoids this issue by mimicking the outline property.

https://github.com/aredotna/ervell/blob/64fc8b4005f5ad68cbc96d01fed5869fd5fbcd36/react/styles/Mixins.js#L5-L17

File: react/components/AddBlock/index.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+const Interior = styled.div`
+  background: white;
+  display: flex;
+  height: calc(100% - 0.5em - 2px);
```
Comment (dzucconi) [inline react/components/AddBlock/index.js]: Why the calc?

File: react/components/AddBlock/index.js (hunk 1 above)
Comment (broskoski) [reply react/components/AddBlock/index.js]: Ah got it.

File: react/components/AddBlock/index.js (hunk 2 above)
Comment (broskoski) [reply react/components/AddBlock/index.js]: Bc of the above alignment thing.

**Comment by dzucconi** [comment]
Looks good to me. I always assume calc is a code smell but it might also be fine so feel free to ignore.

**Pull request closed without merging**